pkg/controllermanager/controller/shoot/quota/reconciler.go

```diff
@@ -44,6 +44,9 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	log := logf.FromContext(ctx)
 
+	ctx, cancel := context.WithTimeout(ctx, r.Config.SyncPeriod.Duration)
```

Comment:
It looks like this field is not defaulted.
I assume this leads to a 0s timeout.

Reply:
As there are other controllers as well where `SyncPeriod` is not defaulted. Hence for now we are using the `DefaultTimeout` which is `1min`. 
Defaulting of `SyncPeriod` should be done in another PR.